examples.md

```diff
@@ -61,7 +61,7 @@ Fast Holographic Deconvolution: A New Technique for Precision Radio Interferomet
 Deconvolution settings   
 ~~~
 deconvolve = 1 
-max_sources = 200000
+max_deconvolution_components = 200000
```

Comment:
I think the value used for the example should match the default value in `general_obs`. That is currently 10,000, though the dictionary says 100,000 and we have 200,000 here. I suggest changing them all to match the value in the dictionary.

Reply:
I see that it is also set in `fhd_init`, though that will always be ignored since this default is set first. We must remove default values from `general_obs` that we actually want set somewhere else.